bundles/org.openhab.binding.max/README.md

```diff
@@ -132,15 +130,31 @@ _Cube device configurable parameters_
 
 ## Action Buttons (visible in HABmin)
 
-In the HABmin thing configuration screen, several action buttons are available to trigger special actions on the Cube
+The Action Buttons are deprecated and will be removed in a future version.
+
+## Thing Actions
+
+Several Thing Actions are available to trigger special actions on the MAX! Cube
 
-*   _Reset Cube Configuration_ resets the MAX! Cube room and device information. Devices will need to be included again! To use this button you need to enable 'Advanced settings'
+*   `reset()`: _Reset Cube Configuration_ resets the MAX! Cube room and device information. Devices will need to be included again!
 
-*   _Restart Cube_ triggers the reboot of a Cube. This can be used if a Cube became unresponsive to commands or no connection can be made. (e.g. if you tried to connect to the Cube with multiple applications at the same time)
+*   `reboot()`: _Restart Cube_ triggers the reboot of a Cube. This can be used if a Cube became unresponsive to commands or no connection can be made. (e.g. if you tried to connect to the Cube with multiple applications at the same time)
 
 On the MAX! devices you can trigger the following action
 
-*   _Delete Device from Cube_ deletes the device from the MAX! Cube. Device will need to be included again! To use this button you need to enable 'Advanced settings'
+*   `deleteFromCube()`: _Delete Device from Cube_ deletes the device from the MAX! Cube. Device will need to be included again!
 
-Note: In Paper UI there are no action buttons.
-You can trigger these actions changing from 'No Action' in the dropdown to the action value.
```

Comment:
This looks like still valid, and the way to trigger the action from the GUI or am I missing something?